### ISSUE TITLE
Fails to find type alias

### ISSUE THREAD

**Issue by weirdan** [vimeo/psalm#904]
https://getpsalm.org/r/787b9e9387
```php
<?php
  /** 
   * @psalm-type bType 'a'|'b'|'c'
   */  
  /**
   * @psalm-return bType 
   */
  function f2() {
    return 'a';
  }
```
**Expected**: no errors
**Actual**: Class or interface bType does not exist
